apps/modernization-ui/src/pages/addPatient/components/LeftBar/LeftBar.tsx

```diff
@@ -22,15 +25,36 @@ export const LeftBar = ({ activeTab }: any) => {
                     New patient
                 </h6>
             </div>
-            {/* To be added new in future a list of other pages, for now its commented out as its not needed until designers have built out designs for these pages */}
-            {/* <div className="border-base-light grid-row flex-no-wrap border-top padding-y-1 margin-x-3 cursor-pointer width-full">
-                <h6
-                    className={`${
-                        activeTab === ACTIVE_TAB.ORGANIZATION && 'active'
-                    } text-normal margin-0 font-sans-md padding-bottom-1 width-full padding-y-1 padding-left-2`}>
-                    New {ACTIVE_TAB.ORGANIZATION}
-                </h6>
-            </div> */}
+            <div className="border-base-light grid-row flex-no-wrap border-top padding-y-1 margin-x-3 cursor-pointer width-full">
```

Comment:
All of these menu items are copied over and over, would be better to create a component to encapsulate a single menu item and use that for each item